foxai/explainer/computer_vision/algorithm/conductance.py

```diff
@@ -173,16 +179,35 @@ def calculate_features(
             layer = get_last_conv_model_layer(model=model)
 
         conductance = self.create_explainer(model=model, layer=layer)
-        attributions = conductance.attribute(
-            input_data,
+
+        attributions_list: List[torch.Tensor] = []
```

Comment:
You could use new AttrobutionType ;)

Reply:
Fixed: https://github.com/softwaremill/FoXAI/pull/97/commits/e62e2b2b737447cb080e63aedf5847054f1d3b1d.